system/Cookie/Cookie.php

```diff
@@ -595,8 +595,8 @@ public function offsetGet($offset)
     /**
      * Offset to set.
      *
-     * @param string $offset
-     * @param mixed  $value
+     * @param string                                  $offset
+     * @param array|bool|float|int|object|string|null $value
```

Comment:
`bool|int|string`?
But, after all, we cannot set anything.

Reply:
Modified.